xml/System.ComponentModel/TimeOnlyConverter.xml

```diff
@@ -151,7 +151,7 @@
         <param name="culture">To be added.</param>
         <param name="value">To be added.</param>
         <param name="destinationType">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Converts the given value object to a <see cref="T:System.TimeOnly" /> object using the arguments.</summary>
```

Comment:
@tarekgh - this seems wrong. It should be "from a <see cref="T:System.TimeOnly" />"

Reply:
The original text looks correct to. This is the `ConvertTo` method which tasks objects (either string or TimeOnly) and return TimeOnly.

Reply:
I stand corrected here. Eric is right. It convert from TimeOnly to object (mostly a string).

Reply:
Thanks. Correcting with a suggestion:

```suggestion
        <summary>Converts the given value object from a <see cref="T:System.TimeOnly" /> object using the arguments.</summary>
```

Reply:
@eerhardt does your suggestion apply to the two summaries that look similar, or just to this one?

Reply:
Looks like the same problem is in DateOnlyConverter.

Should we also update the `///` comments in dotnet/runtime?

Reply:
yes, I'll take care with that.

Reply:
@eerhardt  I opened the PR https://github.com/dotnet/runtime/pull/72569